cpp/CMakeLists.txt

```diff
@@ -1,18 +1,11 @@
+#------------------------------------------------------------------------------
 # Top level CMakeLists.txt file for DOLFIN
 cmake_minimum_required(VERSION 3.10)
```

Comment:
I think this needs to change? I can't now build with cmake 3.10.2

Reply:
Strange, the CMake version in the Docker containers running on CircleCI is 3.10.2.

What's the error message?

Reply:
Sorry, my problem. Now fixed.